assets/styles/global.scss

```diff
@@ -7,14 +7,17 @@ html,
 body {
   margin: 0;
   padding: 0;
-  background-color: #2c2c2c;
+  background-color: var(--cyber-dark-grey);
   color: #ffffff;
   
   --min-nav-inner-width: 8ch;
   --nav-width: calc(max(var(--min-nav-inner-width), 10%) + 2rem);
   --nav-footer-font-size: max(2.1vw, calc(0.21 * var(--min-nav-inner-width)));
   --nav-footer-line-height-multiplier: 1.32;
   --nav-footer-line-height: calc(var(--nav-footer-line-height-multiplier) * var(--nav-footer-font-size));
+  --cyber-yellow: #FFBD3F;
```

Comment:
make this lowercase

Reply:
Fixed in ef2e7e772e9f3db375e85677fc4c88df39c00b1a